__tests__/app.test.js

```diff
@@ -60,3 +49,43 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("200: responds with an article object ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          article_id: 1,
```

Comment:
Good job checking the article id matches, gives you enough confidence that you have the correct article back I feel!